Source/WTF/wtf/PlatformEnable.h

```diff
@@ -549,12 +549,20 @@
 #endif
 
 #if !defined(ENABLE_WEBXR)
+#if PLATFORM(VISION)
```

Comment:
PlatformEnableCocoa?

Reply:
I figured other ports might enable WebXR at some point, so it wasn't really a cocoa-specific flag.

Reply:
Oh! I misunderstood how these layer. Yes, I'll move it.

Reply:
Right, this is the default value for WebKit as a whole, and the platforms override in their children